internal/cephfs/core/volume.go

```diff
@@ -255,6 +255,12 @@ func (s *subVolumeClient) CreateVolume(ctx context.Context) error {
 	if err != nil {
 		log.ErrorLog(ctx, "failed to create subvolume %s in fs %s: %s", s.VolID, s.FsName, err)
 
+		if errors.Is(err, rados.ErrNotFound) {
```

Comment:
Should we also defend CreateSubVolumeGroup() for EEXITS error?

Reply:
No, its idempotent, if the group already exists we will get success response